web/docs/auth/auth-hooks.md

```diff
@@ -433,7 +440,7 @@ Wasp calls the `onAfterLogin` hook after the user logs in.
 
 The `onAfterLogin` hook can be useful if you want to perform some action after the user logs in, like syncing the user with a third-party service.
 
-Since the `onAfterLogin` hook receives the OAuth access token, it can also be used to update the OAuth access token for the user in your database.
+Since the `onAfterLogin` hook receives the OAuth tokens, you can also use it to update the OAuth access token for the user in your database. You can also use it to [refresh the OAuth access token](#refreshing-the-oauth-access-token) if the provider supports it.
```

Comment:
Redundant "also" in the first sentence I think (makes the second sentence seeem repetitive).

Reply:
I already have this version locally, this is some weird Github diff.